selfdrive/car/__init__.py

```diff
@@ -43,7 +43,8 @@ def gen_empty_fingerprint():
 # FIXME: hardcoding honda civic 2016 touring params so they can be used to
 # scale unknown params for other cars
 class CivicParams:
-  MASS = 1326. + STD_CARGO_KG
+  MASS = 1326.
+  MASS_WITH_CARGO = MASS + STD_CARGO_KG
```

Comment:
is MASS only used to set the civic weight in the honda interface? if so, think it's fine to just have that constant twice and delete this MASS

Reply:
The Civic car port [uses three constants](https://github.com/commaai/openpilot/blob/4c1b8b71d096e7658fe56598944e3f8bd3e347f0/selfdrive/car/honda/interface.py#L91-L115) from this class. IMO it doesn't make sense to duplicate the mass and not the other parameters, and having the Civic port use this special snowflake construct seems to have more costs than benefits. LMK how you like the most current version.

Reply:
It also wasn't obvious that changing Honda Civic mass or wheelbase parameters would indirectly alter params for *every other car* known to openpilot.

Perhaps we accept that they're decoupled, and rename the CivicParams class to BenchmarkParams and treat it as a snapshot-in-time of a nonspecific idealized/typical car?

Reply:
We can do that in another PR, will make the change easier to think about

Reply:
https://github.com/commaai/openpilot/pull/29459